apps/website/blog/2024-12-01-maci-getting-started.md

```diff
@@ -0,0 +1,294 @@
+```
+
+:::note
+Unless you are looking to contribute to the MACI codebase, we suggest you use the latest released version. You can check all the releases [here](https://github.com/privacy-scaling-explorations/maci/releases).
+:::
+
+### Download the zero knowledge artifacts
+
+MACI has two main zk-SNARK circuits, and each of them is parameterized. There should be one `.zkey` file for each circuit and set of parameters.
+
+Unless you wish to generate a fresh set of `.zkey` files, you should obtain
+them from someone who has performed a multi-party trusted setup for said
+circuits. For more details on which artifacts have undergone a trusted setup, please refer to the [Trusted Setup](/docs/security/trusted-setup) page.
+
+:::important
+Note the locations of the `.zkey` files cause you will need it when deploying contracts. (put in the deploy-config.json)
+:::
+
+#### Download test artifacts
+
+For all but production use cases, we suggest using the test artifacts, with the latest dev code, you can download them by running:
+
+```bash
+pnpm download-zkeys:test
+```
+
+#### Download ceremony artifacts
+
+For production you need to use the ceremony artifacts which have undergone a trusted setup, you can download them with the command:
+
+```bash
+pnpm download-zkeys:ceremony
+```
+
+:::info
+Currently, the ceremony artifacts work with MACI version up to 2.x
+:::
+
+## Deploy Contracts
+
+Before deploying the contracts we need to do some final configurations to our repository.
+
+### Set the environment variables
+
+Head to the `packages/contracts` folder and copy the `.env.example` file.
+
+```bash
+cd packages/contracts && \
+cp .env.example .env
+```
+
+Make sure to include a mnemonic and RPC url (make sure to replace NETWORK with the network you want to use).
+
+```js
+MNEMONIC = "your ethereum secret key";
+NETWORK_RPC_URL = "the eth provider url";
+NETWORK_ETHERSCAN_API_KEY = "etherscan api key";
+```
+
+### Generate Coordinator Keys
+
+In order to run MACI polls, a coordinator is required to publish their MACI public key. You will need to generate a MACI keypair, and treat the private key just as your ethereum private keys. Please store them in a safe place as you won't be able to finish a round if you lose access, or if compromised a bad actor could decrypt the vote and publish them online. You can generate a new key pair using maci-cli by running the following commands:
+
+```bash
+cd packages/cli && \
+node build/ts/index.js genMaciKeyPair
+```
+
+### Set the configuration file
+
+:::note
+There are already some deployed contracts that could be reused. More information can be found in the (incremental documentation page)[docs/getting-started#deploy-maci-contracts].
+:::
+
+Head back to the contracts folder and copy the config example and update the fields as necessary:
+
+```bash
+cd ../contracts && \
+cp deploy-config-example.json deploy-config.json
+```
+
+#### ConstantInitialVoiceCreditProxy
+
+Defines how many credits will get each voter.
```

Comment:
suggest "Specifies the number of credits allocated to each voter"
also I think we could include the doc link here? `/docs/technical-references/smart-contracts/VoiceCreditProxy`

Reply:
I added a link at the end that sends to our documentation so they can read the rest.